verifier.py

```diff
@@ -417,73 +379,50 @@ def main():
+                    nodes_list_verified += project_dashboards
             if run_info['scrape_registrations']:
                 # Must run all page types automatically
-                registration_files = RegistrationFilesVerifier()
-                registration_wiki = RegistrationWikiVerifier()
-                registration_analytics = RegistrationAnalyticsVerifier()
-                registration_forks = RegistrationForksVerifier()
-                registration_dashboards = RegistrationDashboardVerifier()
-                for url in run_info['registration_urls']:
-                    if not url.endswith("fail"):
-                        run_info['registration_urls'].remove(url)
-                rescrape_list.extend((registration_files.failed_pages + registration_wiki.failed_pages +
-                                      registration_analytics.failed_pages + registration_forks.failed_pages +
-                                      registration_dashboards.failed_pages))
+                registration_files = RegistrationFilesVerifier().failed_pages
+                registration_wiki = RegistrationWikiVerifier().failed_pages
+                registration_analytics = RegistrationAnalyticsVerifier().failed_pages
+                registration_forks = RegistrationForksVerifier().failed_pages
+                registration_dashboards = RegistrationDashboardVerifier().failed_pages
+                registrations_list_verified = registration_files + registration_wiki + registration_analytics + \
+                    registration_forks + registration_dashboards
             if run_info['scrape_users']:
-                user_profiles = UserProfileVerifier()
-                for url in run_info['user_profile_page_urls']:
-                    if not url.endswith("fail"):
-                        run_info['user_profile_page_urls'].remove(url)
-                rescrape_list.extend(user_profiles.failed_pages)
+                user_profiles_verified = UserProfileVerifier().failed_pages
             if run_info['scrape_institutions']:
-                institution_dashboards = InstitutionDashboardVerifier()
-                for url in run_info['institution_urls']:
-                    if not url.endswith("fail"):
-                        run_info['institution_urls'].remove(url)
-                rescrape_list.extend(institution_dashboards.failed_pages)
-    # json.dump[run_info]   ??????
+                institution_dashboards_verified = InstitutionDashboardVerifier().failed_pages
 
-    # Rescraping
+            # clearing json file and dumping in new lists
+            run_info['node_urls'] = nodes_list_verified
+            run_info['registration_urls'] = registrations_list_verified
+            run_info['user_profile_page_urls'] = user_profiles_verified
+            run_info['institution_urls'] = institution_dashboards_verified
+            with codecs.open(TASK_FILE, mode='w', encoding='utf-8') as file:
+                json.dump(run_info, file)
```

Comment:
This part doesn't run as it is ( <type>_list_verified referenced before assignment)